requirements/base.in

```diff
@@ -0,0 +1,6 @@
+# Core requirements for using this package
+-c constraints.txt
+
+django
+edx-opaque-keys
+-e git+https://github.com/edx/xblock-lti-consumer.git@v1.1.8#egg=lti_consumer-xblock==1.1.8
```

Comment:
@morenol please update this req. 
we have released the new [version](https://github.com/edx/xblock-lti-consumer/releases) today.

Reply:
you can make a new PR with this change. I have merged this one.